### ISSUE TITLE
Food for thought

### ISSUE THREAD

**Issue by NavidMitchell** [SpartanJ/efsw#113]
Hello, Thank you for building efsw. It is exactly what we were looking for. 

Since this is intended to be a public project it would be helpful to include semantic versioning or the like. Our team has found this article invaluable for creating a process around git that provides a nice way to manage our versions and the code base.

https://nvie.com/posts/a-successful-git-branching-model/

**Comment by SpartanJ** [comment]
Hi Navid, what exactly would you like to see? This is a small project and it's feature complete in the sense I don't plan to add new features, only fixes, and minor improvements. I kinda forgot to tag a new version for some time, so I just added it. I'm gonna tag more frequently. Regards.

**Comment by NavidMitchell** [comment]
Hello Spartan,
I was not looking for anything specific. Just thought I would share a git model we found useful internally. Since I did not see any branches or tags for quite some time. As for your goals, tagging the repository with versions should be sufficient. I would recommend following the semantic versioning scheme to inform your users about the scope of your changes. 
Thank you again for efsw! I am dropping it in our code base as we speak.
